features/FxA-54-connect-another-device-phase-3/README.md

```diff
@@ -0,0 +1,367 @@
+
+###### Sign up in Firefox desktop, send link to Android, Firefox not installed
+
+1. Using Firefox desktop, sign in to Sync by creating a new Firefox Account using
+   a mozilla or softvision address.
+2. In the same instance of Firefox desktop, open the verification link with
+   the `&signinCodes=true&forceExperimentGroup=treatment` query parameters.
+3. Enter a valid phone number for an Android device, submit.
+4. Open the SMS link on an Android device.
+5. Install Firefox from the Google Play store.
+6. Open Firefox, complete the "firstrun flow".
+7. Firefox Accounts should open, email address from step 1 should be filled in.
+8. Sign in with the password from step 1.
+
+###### Sign up in Firefox desktop, send link to Android, Firefox already installed.
+
+1. Using Firefox desktop, sign in to Sync by creating a new Firefox Account using
+   a mozilla or softvision address.
+2. In the same instance of Firefox desktop, open the verification link with
+   the `&signinCodes=true&forceExperimentGroup=treatment` query parameters.
+3. Enter a valid phone number for an Android device, submit.
+4. Open the SMS link on an Android device.
+5. Firefox should open to the firstrun flow. Complete the "firstrun flow".
+6. Firefox Accounts should open, email address from step 1 should be filled in.
+7. Sign in with the password from step 1.
+
+##### Reference Images
+
+###### Send SMS
+<img src="images/send_sms.png" width="250" alt="Send SMS"/>
+
+###### SMS received
+Image needed
+
+###### Install Firefox
+<img src="images/install_firefox_play_store_android.png" width="250" alt="Send SMS"/>
+
+###### Open Firefox
+<img src="images/open_firefox_play_store_android.png" width="250" alt="Send SMS"/>
+
+###### Firstrun
+Image needed
+
+##### Signin page with valid signinCode
+<img src="images/signin_valid_code_android.png" width="250" alt="Send SMS"/>
+
+#### iOS
+
+##### Testing without support for deep links in Firefox for iOS
+
+Support for deep linking and signinCodes has not yet landed in the release
+channels of Firefox for iOS. Until support lands, we have to
+manually open the deep link.
+
+###### Sign up in Firefox desktop, send link to iOS, Firefox not installed
+
+1. Using Firefox desktop, sign in to Sync by creating a new Firefox Account using
+   a mozilla or softvision address.
+2. In the same instance of Firefox desktop, open the verification link with
+   the `&signinCodes=true&forceExperimentGroup=treatment` query parameters.
+3. Enter a valid phone number for an iOS device, submit.
+4. Open the SMS link on an iOS device. The link will be of the form `https://<server>/m/:signinCode`, write down the `signinCode`.
+5. Safari will open and a dialog will ask `Open this page in "App Store"?` Touch `Open`.
+6. The Apple app store will open to the Firefox page. Click the install button.
+7. Firefox should install. When complete, click `Open`.
+8. Firefox will open to the Firefox for iOS firstrun page. Click `Start Browsing`.
```

Comment:
This is where @ryanfeeley will have improved steps since we have a lot of drop-off from login.
If his improved steps are believed to be scope creep (not sure how well it was communicated), we can do an update after but it will be a step in the right direction to setup phase 4.